analyses/KY_cd_2020/doc_KY_cd_2020.md

```diff
@@ -0,0 +1,23 @@
+# 2020 Kentucky Congressional Districts
+
+## Redistricting requirements
+In Kentucky, under the Criteria and Standards for Congressional Redistricting adopted by Interim Joint Committee on State Government’s Redistricting Subcommittee in 1991, districts must:
+
+1. be contiguous
+2. have equal populations
+3. be geographically compact
+4. preserve county and municipality boundaries as much as possible
+5. preserve communities of interest as much as possible
+
+### Interpretation of requirements
+We enforce a maximum population deviation of 0.5%.
+
+## Data Sources
+Data for Kentucky comes from the ALARM Project's [2020 Redistricting Data Files](https://alarm-redist.github.io/posts/2021-08-10-census-2020/).
+
+## Pre-processing Notes
+No manual pre-processing decisions were necessary.
+
+## Simulation Notes
+We sample 5,000 districting plans for Kentucky.
+To balance county and municipality splits, we create pseudocounties for use in the county constraint, which leads to fewer municipality splits than using a county constraint.
```

Comment:
can we add a little more detail here? See e.g. https://github.com/alarm-redist/fifty-states/blob/main/analyses/MN_cd_2020/doc_MN_cd_2020.md

Reply:
and a sentence up in the 'interpretation' section as well about this. the MN doc should give you a good template.